src/Interpreters/InterpreterKillQueryQuery.cpp

```diff
@@ -451,6 +451,7 @@ AccessRightsElements InterpreterKillQueryQuery::getRequiredAccessForDDLOnCluster
                 | AccessType::ALTER_DELETE
                 | AccessType::ALTER_MATERIALIZE_INDEX
                 | AccessType::ALTER_MATERIALIZE_COLUMN
+                | AccessType::ALTER_MATERIALIZE_COLUMNS
```

Comment:
Why we don't need the same for  `ALTER_MATERIALIZE_INDEXES` and `ALTER_MATERIALIZE_PROJECTIONS`?

Reply:
aha. I missed them. ^^. will add them